### ISSUE TITLE
Patches are not working, and JS Patches break steamwebhelper.exe

### ISSUE THREAD

**Issue by GoldTitaniumSulfurCarbon** [Jonius7/SteamUI-OldGlory#113]
As recent as a couple months ago, the new Steam updates have broken the old features, but for the 5.15 update, it outright does not want to apply anymore, and JS patches gives me the "steamwebhelper.exe is not working" issue from Steam. Before that, it was working just fine, but now, it does not want to apply, and it breaks Steam in the process.

**Comment by OsuCelsius** [comment]
Yeah, same problem, css changes work fine but js breaks steamwebhelper

**Comment by Jonius7** [comment]
It does seem to be broken on Steam Beta, and I still haven't figured out why...

**Comment by Jonius7** [comment]
It's not looking good, it appears Steam now has an invisible check that determines whether the JS has been modified. As in, because OldGlory beautifies the code (and then minifies it), the resulting code is not the same format as the original. So there's too many differences, and bam, Steam won't load. Tiny manual changes won't trigger this though.

**Comment by GoldTitaniumSulfurCarbon** [comment]
> It's not looking good, it appears Steam now has an invisible check that determines whether the JS has been modified. As in, because OldGlory beautifies the JS code (and then minifies it), the resulting code is not the same format as the original. So there's too many differences, and bam, Steam won't load. Tiny manual changes won't trigger this though, but the file is impractical to edit without beautifying it first.

Couldn't there be a way to edit some config file to prevent the JS from being barred or something like that? I recall some time ago, when I was dabbling with different UIs, there was some file that could be deleted in order to prevent Steam from updating, and thus not undoing the "downgrade" done.

**Comment by Jonius7** [comment]
It looks like Steam Families has been released onto stable. There may be a way to still have JS tweaks available by editing the JS files directly (bit-by-bit instead of line-by-line), but I am not sure how reliable this is. As of now, JS tweaks will not be working.

**Comment by sunlightspectre** [comment]
Just got booted off of Steam, saw the news.   
I'd make a joke about how what's more fitting for the 'family' update than a shattered home, or functionality...   
But at the moment, I'd like to know how to access steam.   
I'm now stuck at synchronising and using the 'reset' option - which I figured would restore CSS + JS to their default before I can at least apply the CSS tweaks... 

For now...

Would work.  
Unfortunately, that doesn't seem to be the case?

**Comment by Jonius7** [comment]
> Editing, just wanted to mention I re-installed.
At worst case, you can just delete the `steamui` folder and restart instead of having to reinstall Steam entirely.

At this point I don't use the JS at all.
I'd want to release a new version that doesn't have JS tweaks at all anymore but removing a part of what is basically my life's work hurts

> hand-editing the JS 'stick.'
kinda does work if you don't make any drastic changes
